### ISSUE TITLE
dependencies version not updated in crates

### ISSUE THREAD

**Issue by Song-aff** [almindor/mipidsi#128]
The dependencies in the repository are as follows:
```
[dependencies]
display-interface = "0.5.0"
embedded-graphics-core = "0.4.0"
embedded-hal = "1.0.0"
nb = "1.0.0"
```
They are inconsistent with the dependencies listed on crates.io for version 0.7.1 of the mipidsi crate.

**Comment by rfuest** [comment]
The dependencies in the development version of the crate have been updated to support the 1.0 release of `embedded-hal`. The dependencies on crates.io will be updated with the next release of `mipidsi`.

**Comment by almindor** [comment]
This is now solved with 0.8 out
